Doc/library/tempfile.rst

```diff
@@ -52,6 +52,8 @@ The module defines the following user-callable items:
    stored.  *buffering*, *encoding* and *newline* are interpreted as for
    :func:`open`.
 
+   The default value of the *buffering* parameter is -1, which prevents buffering and is interpreted as for the :func:`open`.
```

Comment:
Shouldn’t buffering=-1 mean a fixed buffer based on OS properties or the _io_ module? To prevent buffering, I would suggest buffering=0.

Reply:
@vadmium, yes I agree. I will make the required change.